src/css/profile/mobile/common/popup.less

```diff
@@ -11,16 +11,25 @@
 	background-color: @color_popup_bg;
 	border-radius: 26 * @px_base;;
 
-	@media (orientation: portrait) and (max-width: 479px) {
+	// mobile
+	@media (min-width: 0px) and (max-width: 479px) {
 		width: 100%;
 	}
-	@media (orientation: landscape) and (max-height: 479px) {
+	@media (min-height: 0px) and (max-height: 479px) {
 		width: 55%;
 	}
-	@media (orientation: portrait) and (min-width: 673px) {
+	//winner
+	@media (min-width: 480px) and (max-width: 672px) and (min-height: 480px) {
+		width: 63%;
+	}
+	@media (min-height: 480px) and (max-height: 672px) and (min-width: 480px) {
+		width: 55%;
+	}
+	//tablet
+	@media (min-width: 673px) and (max-width: 985px) and (min-width: 673px) {
```

Comment:
Why did you separate 
	@media (min-height: 480px) and (max-height: 672px) and (min-width: 480px) {
		width: 55%;
	}

Reply:
done.